browser/permissions/brave_wallet_permission_prompt_android.h

```diff
@@ -39,7 +39,8 @@ class BraveWalletPermissionPrompt
   };
 
   BraveWalletPermissionPrompt(content::WebContents* web_contents,
-                              std::unique_ptr<Delegate> delegate);
+                              std::unique_ptr<Delegate> delegate,
+                              brave_wallet::mojom::CoinType coin_type);
```

Comment:
make #include "brave/components/brave_wallet/common/brave_wallet.mojom.h" for the CoinType.